apps/web/components/PageWrapper.tsx

```diff
@@ -27,6 +30,23 @@ const calFont = localFont({
   weight: "600",
 });
 
+const Redirects = () => {
+  const session = useSession();
+  const router = useRouter();
+  const pathName = usePathname();
+  // is the connected user in a platform organization or not
+  const isPlatformUser = session?.data?.isPlatformUser ?? false;
+
+  useEffect(() => {
+    // is platform user trying to access a non-platform page, redirect to platform dashboard
+    if (isPlatformUser && pathName && !pathName.startsWith("/settings")) {
```

Comment:
```suggestion
    if (!isPlatformUser || !pathName) {
      return
    } 
    
    if(!pathName.startsWith("/settings")) {
```
https://handbook.cal.com/engineering/best-practices/prefer-early-returns